.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,7 @@
+---
+name: 🚀 Feature request
+about: Suggest a new feature for this project
+
+---
+
+Most major feature requests are made over at the [TSC](https://github.com/AdoptOpenJDK/TSC/).
```

Comment:
Maybe "Features that impact the whole project (e.g. Adding a new OpenJDK distribution) are made over...."